.storybook/config.js

```diff
@@ -1,9 +1,30 @@
 import { configure } from '@kadira/storybook';
+import ThemedStyleSheet from 'react-with-styles/lib/ThemedStyleSheet';
+import aphroditeInterface from 'react-with-styles-interface-aphrodite';
 
 import '../css/slider.css';
 import '../css/slider-horizontal.css';
 import '../css/slider-vertical.css';
 
+/* Register react with styles interface */
+ThemedStyleSheet.registerTheme({
+  unit: 30,
+  responsive: {},
```

Comment:
we should probably add the `largeAndAbove` key that we use. It can match what we have in `dls-web`.